src/output.rs

```diff
@@ -78,6 +128,11 @@ fn print_entry_colorized<W: Write>(
     }
 
     if offset > 0 {
+
+        if needs_quoting {
+            offset += 2;
```

Comment:
I don't think this is correct. If it is only quoted shouldn't the offset be just 1 more? 

And if it is double quoted, and has escaped characters, then the proper offset will depend on how many characters in the parent ended up getting escaped.